specs/build.gradle

```diff
@@ -0,0 +1,129 @@
+apply plugin: "groovy"
```

Comment:
@urvashisingh this little guy allows us to run specs from IDEA (But I need com.electriccloud:commander-spec-tests:latest.integration this artifact available somewhere. For now I have placed it onto my local artifactory.)

Reply:
@imago-storm I think we are using bitbucket for the electric-flow gradle plugin. So, we could put this also in the same repo.